.ci/templates/extension-tests-devfile.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: 1.0.0
+metadata:
+  name: extension-test-sample
+components:
+  - mountSources: true
+    memoryLimit: 3Gi
+    type: dockerimage
+    image: 'quay.io/eclipse/che-theia-dev:next'
+    alias: che-dev
+  - type: chePlugin
+    reference: 'https://raw.githubusercontent.com/svor/che-vscode-extension-tests/main/meta.yaml'
```

Comment:
Is there a central place we can put this, i.e. not your fork?

Reply:
@ericwill I think we can move this extension into che-incubator, but when https://github.com/eclipse/che/issues/19172 will be resolved. I'm still working on this extension.